help/en/package/jmri/jmrit/operations/Operations.shtml

```diff
@@ -260,20 +260,29 @@
     
     <a name="Display"></a>
     <h3>Display and Help</h3>
-    
-    To adjust the display font size or appearance of the JMRI windows, select "Preferences.." found under
-    "Edit" in the main window menu bar. A new window will appear with options to make layout connections, change
-    the "Display" appearance and font size, and many other settings including the "Railroad Name". Most
-    JMRI windows have as part of the menu bar the "Help" option. Select "Window Help.." to get information
-    about the window you're using. 
-    
+
```

Comment:
All this section has is links back to preferences help. It is a summary of available features.

 

Dan

 

From: Egbert Broerse [mailto:notifications@github.com] 
Sent: Tuesday, April 26, 2016 2:17 PM
To: JMRI/JMRI <JMRI@noreply.github.com>
Cc: Daniel Boudreau <daboudreau@hotmail.com>; Author <author@noreply.github.com>
Subject: Re: [JMRI/JMRI] Improve help (#1228)

 

In help/en/package/jmri/jmrit/operations/Operations.shtml <https://github.com/JMRI/JMRI/pull/1228#discussion_r61137072> :

> @@ -260,20 +260,29 @@
>      
>      <a name="Display"></a>
>      <h3>Display and Help</h3>
> -    
> -    To adjust the display font size or appearance of the JMRI windows, select "Preferences.." found under
> -    "Edit" in the main window menu bar. A new window will appear with options to make layout connections, change
> -    the "Display" appearance and font size, and many other settings including the "Railroad Name". Most
> -    JMRI windows have as part of the menu bar the "Help" option. Select "Window Help.." to get information
> -    about the window you're using. 
> -    
> +

It would be even better to add this to the Preferences help <http://jmri.sourceforge.net/help/en/package/apps/TabbedPreferences.shtml>  and just add a link from this page. That way, we don't have to maintain the same info in two places.

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub <https://github.com/JMRI/JMRI/pull/1228/files/ac585d7eac72b5ba77be6b34ce3d4a26303e1e74#r61137072>   <https://github.com/notifications/beacon/ANw2p4nMf_WE_c3DfGqbW0PiKExBMZnqks5p7largaJpZM4IQNJV.gif>